scripts/snippets/inf-hosp.r

```diff
@@ -0,0 +1,19 @@
+## first, choose random delays
+inf_hosp <- infection_times |>
+  mutate(
+    incubation_period = rgamma(n(), shape = 5, rate = 1),
+    onset_to_hosp = rlnorm(n(), meanlog = 1, sdlog = 1),
+    infection_date = as.Date(start_date + infection_time),
+    onset_date = as.Date(start_date + infection_time + incubation_period),
```

Comment:
If infection time discrete? It needs to be continuous for double censoring to be present

Reply:
it's continuous